config.py

```diff
@@ -16,6 +31,11 @@ def getenv(env_var, default=NOT_PROVIDED):
         raise Exception(f"Environment Variable '{env_var}' not set and no default provided")
     return value
 
+def getenv_unique_list(var_name) -> set[str]:
+    l = getenv(var_name).split(" ")
+    s = set(l)
+    assert (len(l) == len(s)), f"{var_name} has duplicate members! {s}"
```

Comment:
hm, error about duplicates should probably show the duplicated list, not the set:
```suggestion
    assert (len(l) == len(s)), f"{var_name} has duplicate members! {l}"
```